Controller.cpp

```diff
@@ -49,7 +55,7 @@ void BeginRecording()
     // Create file name
     time_t t = time(NULL);
     tm* localTime = (localtime(&t));
-    char outputFile[4096]; // TODO : Replace this with platform-appropriate consts
+    char outputFile[4096]; // TODO : Replace this with platform-appropriate consts (-c -i jack,system -o outputFile.wav)
```

Comment:
Ah, I was wondering what the full command line was. I'm keeping this in the header file now (so that it's easy to find and edit). I'll update it there.